pkg/gobmpsrv/gobmpsrv.go

```diff
@@ -77,7 +79,7 @@ func (srv *bmpServer) bmpWorker(client net.Conn) {
 	parserQueue := make(chan []byte)
 	parsStop := make(chan struct{})
 	// Starting parser per client with dedicated work queue
-	go parser.Parser(parserQueue, producerQueue, parsStop)
+	go parser.Parser(parserQueue, producerQueue, parsStop, srv.bmpRaw)
```

Comment:
not great idea changing this function parameters with a one publisher specific parameter, please find a way to pass this parameter by other means